domino/core/pom.xml

```diff
@@ -51,8 +56,7 @@
 							com.ibm.commons;bundle-version="[9.0.0,10.0.0)",
 							org.javolution.core-java;bundle-version="[6.1.0,7.0.0)",
 							org.openntf.formula;bundle-version="1.5.0";visibility=reexport
-						</Require-Bundle>
--->
+ 						</Require-Bundle>
```

Comment:
I'm not happy about using "Require-Bundle" because it is deprecated. But I think there is no way around, because the Domino OSGI-Implementation is very old